_studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h

```diff
@@ -942,6 +942,15 @@ namespace MfxHwH264Encode
 
     ENCODE_FRAME_SIZE_TOLERANCE ConvertLowDelayBRCMfx2Ddi(mfxU16 type, bool bTCBRC);
 
+    enum SliceDividerType
```

Comment:
well, if you are introducing new enum, let's make it scoped enum to avoid implicit conversions:
```
enum class SliceDividerType
{
...
};
```

Reply:
Done.